modules/apps/portal-language/portal-language-override-test/src/testFunctional/tests/LanguageOverride.testcase

```diff
@@ -171,6 +171,50 @@ definition {
 		}
 	}
 
+	@description = "The testcase asserts the override filtered by 'Selected Language'."
+	@priority = "4"
+	test CanFilterOverridesbySelectedLanguage {
+		task ("Given three new keys with two language translations") {
+			LanguageOverride.addLanguageKey(
+				languageKey = "text-contains-error1",
+				translationOverrideList = "en-US",
+				translationOverrideValueList = "Text contains error1");
+
+			LanguageOverride.addLanguageKey(
+				languageKey = "text-contains-error2",
+				translationOverrideList = "en-US",
+				translationOverrideValueList = "Text contains error2");
+
+			LanguageOverride.addLanguageKey(
+				languageKey = "text-contains-error3",
+				translationOverrideList = "es-ES",
+				translationOverrideValueList = "El texto contiene error3");
+		}
+
+		task ("When the user filters by 'Selected Language'") {
+			ManagementBar.setFilterAndOrder(filterBy = "Selected Language");
+		}
+
+		task ("Then the keys should appear and list en-US languages it has translations for") {
+			LexiconEntry.viewEntryName(rowEntry = "text-contains-error1");
+
+			AssertTextEquals(
+				locator1 = "LanguageOverride#LANGUAGE_KEY_OVERRIDDEN_LABEL",
+				value1 = "Languages With Override: en_US");
+
+			LexiconEntry.viewEntryName(rowEntry = "text-contains-error2");
+
+			AssertTextEquals(
+				locator1 = "LanguageOverride#LANGUAGE_KEY_OVERRIDDEN_LABEL",
+				value1 = "Languages With Override: en_US");
+
+			AssertTextNotPresent(
+				key_languageKey = "text-contains-error3",
```

Comment:
Could you double check if the _LexiconEntry.viewNoEntry_ works here? Thanks

Reply:
done